scripts/entity/components/input_components/player_input_component.gd

```diff
@@ -5,6 +5,7 @@ class_name PlayerInputComponent extends BaseInputComponent
 const JUMP_INPUT_NAME = "jump"
 const LEFT_INPUT_NAME = "left"
 const RIGHT_INPUT_NAME = "right"
+const ATTACK_INPUT_NAME = "attack"
```

Comment:
These all should be StringNames. The top three should be fixed after conflicts resolved